index.js

```diff
@@ -95,8 +95,11 @@ module.exports = function(timeout, filter) {
      */
     getHandler: function () {
       var sequence = Array.prototype.slice.call(arguments);
-      return function(files, done) {
-        enqueue.apply(null, sequence.concat(done));
+      return function (files, done) {
+        if (typeof done === 'function' || typeof done === 'string') {
```

Comment:
@nuba From what I can see of the latest `gulp-watch` it only ever calls with a [single `file` argument](https://github.com/floatdrop/gulp-watch/blob/master/index.js#L139).

The two arguments I have are a hangover from a previous version (1.0.7) where `done` may be a [`function` as a result of using `gulp-batch`](https://github.com/floatdrop/gulp-watch/blob/v1.0.7/index.js#L37). So your detection of `function` allows both versions of `gulp-watch` to be used. I think it should be included (as you have done) for correctness but I am not sure it serves a practical purpose.

However I do not see how `done` could be a `string` value. Is this just defensive programming?